### PR TITLE
Simplify model class namespaces.

### PR DESCRIPTION
**Pull request by jander-msft**
Since the functional tests project already has a project-to-project reference to the web api project, there is no need to include links to the model classes in the web api project and the namespaces do not need to be conditional anymore.

### CODE REVIEW

**Review by kkeirstead — APPROVED** [review]
From what I can tell, what's here seems to look good (and tests are passing). I can Approve this, but if you want to get another set of eyes on it before merging maybe @kelltrick can look over it as well.

**Review by hoyosjs — APPROVED** [review]

**Pull request merged**
